packages/webapp/src/components/Task/PureIrrigationTask/styles.module.scss

```
@@ -0,0 +1,33 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+@import '../../../assets/mixin.scss';
+
+.waterCalculatorWrapper {
+  display: flex;
+  gap: 10px;
+  margin-top: 4px;
+
+  @include xs-breakpoint {
```

```suggestion
  @include md-breakpoint {
```
I think I prefer the wrapping right from the tablet breakpoint (and it will probably make even more sense with the new languages coming in this month) -- does this breakpoint look okay to you too?